src/app/pages/ArticlePage/ElectionBanner/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
+import { RequestContext } from '#app/contexts/RequestContext';
+import AmpIframe from '#app/components/AmpIframe';
+import useToggle from '#app/hooks/useToggle';
+import { Tag } from '#app/components/Metadata/types';
+import isLive from '#app/lib/utilities/isLive';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles from './index.styles';
+import BANNER_CONFIG from './config';
+
+export default function ElectionBanner({ aboutTags }: { aboutTags: Tag[] }) {
+  const { service } = useContext(ServiceContext);
+  const { isAmp, isLite } = useContext(RequestContext);
+  const { enabled: electionBannerEnabled }: { enabled: boolean | null } =
+    useToggle('articleElectionBanner');
+
+  if (isLive()) return null; // TODO: Remove once going Live
+  if (isLite) return null;
+
+  const { iframeHeight, iframeSrc, iframeSrcAmp, iframeTitle, thingIds } =
+    BANNER_CONFIG;
+
+  const validAboutTag = aboutTags?.some(tag => thingIds.includes(tag.thingId));
+
+  const showBanner = validAboutTag && electionBannerEnabled;
+
+  if (!showBanner) return null;
+
+  if (isAmp) {
+    return (
+      <div
+        data-testid="election-banner-amp"
```

Comment:
Do we need to have a separate test ID for amp vs canonical? This would simplify your tests a little bit too?

Reply:
Good point, they shouldn't render at the same time so the ID can be the same, shall change!